docs/targets/solana.rst

```diff
@@ -452,6 +452,14 @@ and `system_instruction.spec.ts <https://github.com/hyperledger/solang/blob/main
 
 .. _account_management:
 
+Minimum balance
++++++++++++++++
```

Comment:
This section is misplaced. `.. _account_management:` should be strictly above `Solana Account Management` section.

Reply:
Good point, I don't know how I missed that. Fixed